sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc

```diff
@@ -202,17 +213,31 @@ may only be submitted to a queue that shares the same context.
 The bundle's set of associated devices is the set of devices contained in
 `ctxt`.
 
+Each source language `lang` specifies whether the language is text format or
+binary format, and the application must use the overload that corresponds to
+that format.
+Applications must use overload (1) when the source language is text format and
+must use overload (2) when the source language is binary format.
+The `sycl` language is text format, so application must use overload (1) when
+when creating a kernel bundle from this language.
```

Comment:
```suggestion
when creating a kernel bundle from this language.
```
Double `when` here, as it is already present at the end of the previous line.

Reply:
Thanks.  Fixed in e69a5dda2ef6085a53a43c3daa161c0c7228eab0.